modules/cluster-autoscaler-crd.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * masters/using-cluster-autoscaler.adoc
+
+[id='cluster-autoscaler-crd-{context}']
+= Default Cluster Autoscaler CRD
+
+The following Custom Resource Definition (CRD) shows the parameters and sample
+values for the Cluster Autoscaler.
+
+
+[source,yaml]
+----
+apiVersion: "autoscaling.openshift.io/v1alpha1"
+kind: "ClusterAutoscaler"
+metadata:
+  name: "default"
+spec:
+  podPriorityThreshold: -10 <1>
+  resourceLimits:
+    maxNodesTotal: 24 <2>
+    cores:
+      min: 8 <3>
+      max: 128 <4>
+    memory:
+      min: 4 <5>
+      max: 256 <6>
+    gpus:
+      - type: nvidia.com/gpu <7>
+        min: 0 <8>
+        max: 16 <9>
+      - type: amd.com/gpu <7>
+        min: 0 <8>
+        max: 4 <9>
+  scaleDown:
```

Comment:
We've recently added the `unneededTime` option in the `scaleDown` section. This sets the `--scale-down-unneeded-time` option on the autoscaler, which controls the amount of time before an "unneeded" node will be removed. Still need to update our example.

https://github.com/openshift/cluster-autoscaler-operator/pull/37